test/compilable/ddoc17697.d

```diff
@@ -0,0 +1,15 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS: -D -Dd${RESULTS_DIR}/compilable -o-
+// POST_SCRIPT: compilable/extra-files/ddocAny-postscript.sh 17697
+
+/***
+ * See:
+ *    http://www.fooa.com/test1
```

Comment:
That is a test case for that. Without the change, `test1` would get highlighted in the midst of the URL.